invenio_communities/utils.py

```diff
@@ -109,3 +112,15 @@ def on_datastore_post_commit(sender, session):
             users = role.users.all()
             for user in users:
                 on_user_membership_change(Identity(user.id))
+
+
+def humanize_byte_size(size):
+    """Converts bytes to largest unit (e.g., KB, MB, GB)."""
+    s = Decimal(size)
+    for unit in ["B", "KB", "MB", "GB", "TB", "PB"]:
```

Comment:
It's probably unrealistic that anyone is going to upload anything that is even a gigabyte, let alone a terabyte or petabyte, but I don't think it adds a lot of overhead or anything, so it's likely fine.